src/status_im2/contexts/onboarding/new_to_status/view.cljs

```diff
@@ -93,15 +99,16 @@
   []
   (let [{:keys [top]} (safe-area/get-insets)]
     [:<>
-     [background/view true]
      [rn/view {:style style/content-container}
       [navigation-bar/navigation-bar
-       {:top                   top
-        :right-section-buttons [{:type                :blur-bg
-                                 :icon                :i/info
-                                 :icon-override-theme :dark
-                                 :on-press            #(rf/dispatch
-                                                        [:show-bottom-sheet
-                                                         {:content getting-started-doc
-                                                          :shell?  true}])}]}]
+       {:stack-id                      :new-to-status
+        :override-left-button-on-press navigate-back
```

Comment:
hmm, I really don't like this prop :/ I think it's also a matter that the nav bar needs to be reworked as well, but here can we just pass in the left button props instead of this single override?

Reply:
Done